### ISSUE TITLE
enigma plugin doesn't work in mobile view 

### ISSUE THREAD

**Issue by phonon112358** [messagerie-melanie2/Roundcube-Plugin-Mobile#32]
I have just installed the mobile plugin... great work!!!! ;)

However, when I tried to create pgp keys on the mobile view, the following error is shown:

```
[error]


FILE NOT FOUND
The requested resource was not found!
Please contact your server-administrator.

Failed request:
//192.168.1.124/webmail/?_task=settings&_action=plugin.enigmakeys
```

When I change back to desktop view everything works! 

since I am rather new to roundcubemail (this is my first plugin installation), I have no idea at all why this error appears... 

can you help me?

**Comment by tpayen** [comment]
Hi,
You have to add enigma melanie2_larry_mobile skin in skins folder of your enigma plugin, you can find plugins skin here : https://github.com/messagerie-melanie2/Roundcube-Plugin-Mobile/tree/master/plugins

**Comment by phonon112358** [comment]
I had done that already...!! 
I get only an error when I want to create pgp keys... all other functions of roundcubemail do work greatly in the mobile view...!

**Comment by phonon112358** [comment]
ok... you were right...! I just have copied the files again and now it works! ;)

However, I still miss the function to create/add new pgp keys... 
in the desktop view, there is a `+` symbol at the button. but I see no such thing in the mobile view... (perhaps I oversee it??)

**Comment by ghost** [comment]
This doesn't seem to work any more on 1.3

**Comment by wayneoutthere** [comment]
yeah.  i tried to add skin into enigma skin directory and it didn't work. but, i noticed that the error is asking for 'plugin.enigmakeys' but there is no 'enigmakeys' in any of the enigma directory that I could see...   wish I was smarter :(  I"m using the latest of everything
